# Request
---

# Response
sensor_msgs/Image rgb_image
sensor_msgs/Image depth_image
std_msgs/Float64 timestamp
